SwiftGen.xcodeproj/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>FILEHEADER</key>
+    <string>
+// SwiftGen
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//</string>
```

Comment:
Not really related to this PR, but this special `IDETemplateMacros.plist` file is a great trick to tell Xcode what file header to use when creating new files in this Xcode project. Makes the life of every contributor easier (before that they had to copy/paste and replace the header manually after creating a new file otherwise SwiftLint would warn about bad header, now the right header is already present)

Reply:
TIL